app/boards/intel_adsp_cavs25_tgph.conf

```diff
@@ -74,3 +74,4 @@ CONFIG_DMA_DW_FIFO_PARTITION=y
 CONFIG_DMA_INTEL_ADSP_GPDMA_HAS_LLP=y
 
 CONFIG_HEAP_MEM_POOL_SIZE=8192
+CONFIG_LOG_TIMESTAMP_64BIT=y
```

Comment:
How about changing the default in Kconfig instead?

Reply:
it is shared by all vendors, so I do it only for intel

Reply:
Intel is the only one using IPC4 for now and I see nothing Intel-specific so far so please just change  the default in the KConfig file unless there is a specific reason not to have this by default.

Reply:
I see NXP is working on IPC4 support and reported bugs about intel special code in framework.

Reply:
Why would NXP and others not want this with IPC4? Please explain.

Reply:
Why they can support it ? or need it ?   It needs 64 bits hardware clock support. I am not familiar with other vendor hardware so I can't do it